src/main/java/pro/sky/java/course2/employeelist/controller/EmployeeController.java

```diff
@@ -0,0 +1,42 @@
+package pro.sky.java.course2.employeelist.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.java.course2.employeelist.Employee;
+import pro.sky.java.course2.employeelist.service.EmployeeService;
+
+@RestController
+@RequestMapping ("/employee")
+
+public class EmployeeController {
+
+    private final EmployeeService employeeService;
+
+    public EmployeeController(EmployeeService employeeService) {
+
+        this.employeeService = employeeService;
+    }
+
+    @GetMapping ("/add")
+    public String add (@RequestParam String firstName, @RequestParam String lastName) {
+        Employee result = employeeService.add (firstName, lastName);
+        return getMessage (result, "successfully added");
```

Comment:
```suggestion
        return getMessage(result, "successfully added");
```
 рекомендую использовать ctrl+alt+L для автоформатирования кода